app/src/test/kotlin/com/wire/android/framework/FakeKaliumFileSystem.kt

```diff
@@ -0,0 +1,79 @@
+package com.wire.android.framework
+
+import com.wire.android.config.TestDispatcherProvider
+import com.wire.kalium.logic.data.asset.AssetsStorageFolder
+import com.wire.kalium.logic.data.asset.CacheFolder
+import com.wire.kalium.logic.data.asset.DataStoragePaths
+import com.wire.kalium.logic.data.asset.KaliumFileSystem
+import kotlinx.coroutines.withContext
+import okio.Path
+import okio.Path.Companion.toPath
+import okio.Sink
+import okio.Source
+import okio.buffer
+import okio.fakefilesystem.FakeFileSystem
+import okio.use
+
+class FakeKaliumFileSystem(
```

Comment:
❤️

Reply:
I'm happy to see Fakes as well, it's important and nice we understand the differences from other test patterns in the team 💪